lib/calendarium-romanum/calendar.rb

```diff
@@ -75,6 +75,13 @@ def ==(obj)
       year == obj.year
     end
 
+    def [](args)
+      if(args.is_a?(Range))
+        args.map{|date| day(date)}
+      else
+        day(*args)
+      end
+    end
```

Comment:
Please add [a single blank line](https://github.com/bbatsov/ruby-style-guide#empty-lines-between-methods) after the method.